jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/TestProject.java

```diff
@@ -75,7 +75,11 @@ protected void before() throws Throwable {
     projectRoot = newFolder().toPath();
     copyProject(testProjectName, projectRoot);
 
-    gradleRunner = GradleRunner.create().withProjectDir(projectRoot.toFile()).withPluginClasspath();
+    gradleRunner =
+        GradleRunner.create()
+            .withGradleVersion(JibPlugin.GRADLE_MIN_VERSION.getVersion())
```

Comment:
Should we have parameterized this to run on gradle_min_version and maybe 5,6?

Reply:
That's a good idea. One caveat is that it will double (or triple) the time to run the tests.